.snippets/code/precompiles/registry/PrecompileRegistry.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity >=0.8.3;
+
+/// @author The Moonbeam Team
+/// @title Precompile Registry
+/// @dev Interface to the set of available precompiles.
+interface PrecompileRegistry {
```

Comment:
Please add the info on this PR -> https://github.com/PureStake/moonbeam/pull/2303